exercises/node-javascript/add-esdk-complete/store.js

```diff
@@ -0,0 +1,83 @@
+/* ADD-ESDK-START:
```

Comment:
@mattsb42-aws @lizroth Here is the complete store file.  The esdk has been added.